src/Talis/Persona/Client/Base.php

```diff
@@ -55,40 +57,50 @@
     private $defaultTtl;
 
     /**
+     * @var string
+     */
+    private $appUserAgent;
+
+    /**
      * Constructor
      *
+     * @param string $appUserAgent Consuming application user agent string @since 2.0.0
+     *      examples: rl/5.2, rl, rl/5, rl/5.2 (linux/2.5)
      * @param array $config An array of options with the following keys: <pre>
```

Comment:
if this config object contains optional and mandatory parameters why cant the appuseragent be specified in the config params.